### PR TITLE
content edits

### PR DESCRIPTION
**Pull request by lindsey-rix**

### CODE REVIEW

File: src/app/api/index.html [hunk 1]
```diff
@@ -4,21 +4,11 @@
...
-    </p>
 
-  <stache-page-anchor>
-    Section Two
-  </stache-page-anchor>
-    <p>
-      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Omnis facilis veritatis dicta adipisci, hic, explicabo dignissimos ducimus eligendi, modi odio commodi fugit iusto optio totam corrupti expedita sequi atque assumenda.
-    </p>
+
+
```
Comment (Blackbaud-BrandonJones) [inline src/app/api/index.html]: These extra lines need to be removed.

**Comment by lindsey-rix** [comment]
removed extra lines

File: src/app/index.html [hunk 2]
```diff
@@ -3,10 +3,10 @@
   overlayOpacity="0.4"
   backgroundImageUrl="~/assets/hero-sample-1.jpg">
   <stache-hero-heading>
-    Stache
+      Stache<sup><small>2</small></sup>
```
Comment (Blackbaud-BrandonJones) [inline src/app/index.html]: Not loving the exponent 2,  let's just leave it as Stache 2 for right now (regular 2).

File: src/app/index.html [hunk 3]
```diff
@@ -15,7 +15,7 @@
   showBreadcrumbs="false"
   pageTitle="Welcome">
   <p>
-    Welcome to the SKY UX Stache template.  This template is dynamically cloned anytime <stache-code>skyux new --template stache</stache-code> is run.
+    Welcome to the SKY UX Stache<sup>2</sup> template.  This template is dynamically cloned anytime <stache-code>skyux new --template stache</stache-code> is run.
```
Comment (Blackbaud-BrandonJones) [inline src/app/index.html]: Same as above.

File: src/app/index.html [hunk 4]
```diff
@@ -24,23 +24,23 @@
...
+          For more information on SKY UX, including components and how to manage the builder, see the <a href="https://developer.blackbaud.com/skyux2/">SKY UX docs</a>.
         </p>
     </stache-column>
     <stache-column screenSmall="12" screenMedium="4" screenLarge="4">
       <stache-page-anchor>
         Stache
       </stache-page-anchor>
         <p>
-          For tutorials, information, and recomendations using Stache to create your documentation, please refer to the <a href="https://github.com/blackbaud/stache2-docs">Stache Docs</a>.
+          For more information on how to use Stache to create your documentation, see the <a href="https://github.com/blackbaud/stache2-docs">Stache<sup>2</sup> Docs</a>.
```
Comment (Blackbaud-BrandonJones) [inline src/app/index.html]: and here.

File: src/app/index.html (hunk 2 above)
Comment (lindsey-rix) [reply src/app/index.html]: I removed the formatting. All are regular 2s.

Comment (Blackbaud-BrandonJones) [reply src/app/index.html]: 👍

**Review by Blackbaud-BrandonJones — APPROVED** [review]
Looks awesome! Merge away.

**Pull request merged**